api/v1/storagecluster_types.go

```diff
@@ -137,11 +140,41 @@ type ManageCephObjectStoreUsers struct {
 	ReconcileStrategy string `json:"reconcileStrategy,omitempty"`
 }
 
+// ExternalStorageKind specifies a kind of the external storage
+type ExternalStorageKind string
+
+const (
+	// KindOcs specifies a "ocs" kind of the external storage
+	KindOcs ExternalStorageKind = "ocs"
+
+	// KindRhcs specifies a "rhcs" kind of the external storage
+	KindRhcs ExternalStorageKind = "rhcs"
+)
+
 // ExternalStorageClusterSpec defines the spec of the external Storage Cluster
-// to be connected to the local cluster
+// to be connected to the external cluster
```

Comment:
I think the use of `local` was correct here.

Reply:
The earlier sentence was correct. It refers to the local StorageCluster which will connect to the external ceph cluster

Reply:
done